src/components/sitepages/SiteTargets.vue

```diff
@@ -459,9 +682,31 @@ export default {
             set(val) { this.$store.commit('command_params/mount_object', val)},
         },
 
+        // Common Target computed
+        offset() {
+            return new Date(this.dateobsreal).getTimezoneOffset()
+        },
+        observatoryoffset() {
+            if (this.selected_target_obs =='X') {
+                return this.customobservatoryoffset*60
+            } else {
+                return moment.utc(this.dateobs).tz(this.observatorytime).utcOffset()
+            }
+        },
+        dateobsutc() {
+            return moment(this.dateobsreal).add(this.offset, 'm').toDate()
+            //Start time of observation date in "UTC" (ignore timezone info in moment obj)
+        },
+        dateobsobs() {
+            return moment(this.dateobsreal).add(this.offset+this.observatoryoffset, 'm').toDate()
+            //Start time of observation date in "observatory time" (ignore timezone info in moment obj)
+        },
```

Comment:
these should be snake case too. they are used the same as variables defined in data().